app/models/locker_application.rb

```diff
@@ -69,4 +69,12 @@ def self.search(uid:, archived:)
 
     where(complete: true)
   end
+
+  def self.migrate_accessible_field
+    LockerApplication.where(accessible: true).find_each do |application|
```

Comment:
I believe the accessibility info is used only when assigning lockers.  So, if we don't want to touch as many rows in the migration, we could just select applications that don't yet have an assignment.